server/src/core/datasets/types/dataset.ts

```diff
@@ -0,0 +1,35 @@
+export type DatasetStatistics = {
+  id: number
+  dataset_id: number
+  locale_id: number
+  total_clips_duration: number
+  valid_clips_duration: number
+  average_clips_duration: number
+  total_users: number
+  size: number
+  checksum: string
+  release_date: string
+  name: string
+  release_dir: string
+  download_path: string
+  splits: DatasetSplits
```

Comment:
I was wondering if the `splits` field should be optional since it's possible for some datasets not to have it.